gpd/gpd-pocket/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib, pkgs, ... }:
+
+# GPD Pocket support
+#
+# Most of the configuration comes from the Arch Linux Wiki:
+# https://wiki.archlinux.org/index.php/GPD_Pocket
+#
+# Missing:
+# - media keys
+# - brightness control
+# - video over USB-C
+
+{
+  imports = [
+    ../../common/cpu/intel
+    ../../common/pc/laptop
+  ];
+
+  # kernel
+  # TODO: use https://github.com/jwrdegoede/linux-sunxi?
+  boot = {
+    kernelParams = ["fbcon=rotate:1"];
+    kernelPackages = pkgs.linuxPackages_4_14;
+  };
+
+  # wifi
+  nixpkgs.config.allowUnfree = true;
+  hardware.firmware = with pkgs; [
+    brcmfmac4356-firmware
+  ];
+
+  # power management
+  services.tlp = {
+    enable = true;
+    extraConfig = ''
+      DISK_DEVICES="mmcblk0"
+      DISK_IOSCHED="deadline"
+    '';
+  };
+
+  hardware.pulseaudio = {
+    enable = true;
+    extraConfig = ''
+      set-card-profile alsa_card.platform-cht-bsw-rt5645 HiFi
+      set-default-sink alsa_output.platform-cht-bsw-rt5645.HiFi__hw_chtrt5645_0__sink
+      set-sink-port alsa_output.platform-cht-bsw-rt5645.HiFi__hw_chtrt5645_0__sink [Out] Speaker
+    '';
+    daemon.config = {
+      realtime-scheduling = "no";
+    };
+  };
+
+  # TODO: fix media keys
+  # sound.mediaKeys.enable = true;
```

Comment:
`sound.mediaKeys.enable` overlaps with some desktop managers (e.g. Xfce), which is why it is not set in this repo even if the device does have media keys.